tests/login.spec.js

```diff
@@ -0,0 +1,28 @@
+const { test, expect } = require('@playwright/test');
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://www.saucedemo.com/');
+});
+
+test('login', async ({ page }) => {
+    await page.goto('https://www.saucedemo.com/');
+
+    //login
+    await page.getByPlaceholder('Username').click();
```

Comment:
You don't need to click the field before filling it with value. 
You can delete this line and 13th as unnesessary